src/Compilers/CSharp/Test/Symbol/Symbols/PartialPropertiesTests.cs

```diff
@@ -3627,6 +3750,125 @@ partial class C
             AssertEx.Equal([], property.GetAttributes().ToStrings());
         }
 
+        [Fact]
+        public void PropertyInitializer()
+        {
+            var source = """
+                partial class C
+                {
+                    public partial string P1 { get; set; } = "a";
+                    public partial string P1 { get => ""; set { } }
+
+                    public partial string P2 { get; set; }
+                    public partial string P2 { get => ""; set { } } = "b";
+
+                    public partial string P3 { get; set; } = "c";
+                    public partial string P3 { get => ""; set { } } = "d";
+                }
+                """;
+
+            var comp = CreateCompilation(source);
+            comp.VerifyEmitDiagnostics(
+                // (3,27): error CS8050: Only auto-implemented properties can have initializers.
+                //     public partial string P1 { get; set; } = "a";
+                Diagnostic(ErrorCode.ERR_InitializerOnNonAutoProperty, "P1").WithLocation(3, 27),
+                // (7,27): error CS8050: Only auto-implemented properties can have initializers.
+                //     public partial string P2 { get => ""; set { } } = "b";
+                Diagnostic(ErrorCode.ERR_InitializerOnNonAutoProperty, "P2").WithLocation(7, 27),
+                // (9,27): error CS8050: Only auto-implemented properties can have initializers.
+                //     public partial string P3 { get; set; } = "c";
+                Diagnostic(ErrorCode.ERR_InitializerOnNonAutoProperty, "P3").WithLocation(9, 27),
+                // (10,27): error CS8050: Only auto-implemented properties can have initializers.
+                //     public partial string P3 { get => ""; set { } } = "d";
+                Diagnostic(ErrorCode.ERR_InitializerOnNonAutoProperty, "P3").WithLocation(10, 27));
+        }
```

Comment:
>} [](http://example.com/codeflow?start=8&length=1)

Are we binding the initializers in these error cases? Consider adding a test such as:
```csharp
partial string P4 { get; set; } = nameof(Unknown);
partial string P4 { get => ""; set { } }
```